bixby-views/code/ShowVideo.js

```diff
@@ -0,0 +1,3 @@
+module.exports.function = function showVideo () {
+  return 'https://sample-videos.com/video123/mp4/720/big_buck_bunny_720p_30mb.mp4'
```

Comment:
Just watched this video and can we pick a friendlier one? lol

Reply:
Or use the 1mb version:
https://www.sample-videos.com/video123/mp4/720/big_buck_bunny_720p_1mb.mp4